lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -400,6 +404,17 @@ namespace Js
         return ret;
     }
 
+    template <typename T>
+    static _Ret_notnull_ T AssertNotNullOrOOM(_In_ T value)
```

Comment:
This function does not assert?

Reply:
I have a mild preference for the ContextAPIWrapper approach that JSRT uses

Reply:
It still doesn't assert ?